### DISCUSSION TITLE
PDF document size

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1065] [Help]
```
	MagickReadSettings settings = new MagickReadSettings();
			settings.Width = 3691;
			settings.Height = 2775;
			settings.Density = new Density(300);
			
			using (MagickImage image = new MagickImage("xc:"+"#07553B"+"\"", settings))
			{
			
				image.Format = MagickFormat.Pdf;
				image.Write("cover1.pdf");
			}
```

When i created with ImageMagick - size of document was small
Now with this code size of document more 30Mb!!!
Any suggestions to reduce size?

**Comment by dlemstra** [comment]
What was the version of ImageMagick that you used and what version of Magick.NET are you using? And what is the command that you executed?

**Comment by jaka1-newbie** [comment]
C:\Program Files\ImageMagick-7.0.10-Q16-HDRI\magick.exe
convert -size {-Variable.widthCover-}x{-Variable.heightCover-}  -set colorspace sRGB  xc:"#07553B" basic.jpg

**Reply by jaka1-newbie** [reply]
Magick.Net - 8.3.3.  Magick.NET-Q16-HDRI-AnyCPU

**Reply by dlemstra** [reply]
` basic.jpg` is a JPEG file and not a PDF file?

**Comment by jaka1-newbie** [comment]
I need pdf.

When i change this line:
				`image.Format = MagickFormat.Pdf; 
`
to 
				`image.Format = MagickFormat.Png;`
all gone OK!

**Reply by jaka1-newbie** [reply]
from pdf doc 30Mb to  pdf doc 200-300Kb

**Reply by dlemstra** [reply]
What happens when you change the compression of the image inside the PDF document to Zip: `image.Compression = CompressionMethod.Zip;`?

**Reply by jaka1-newbie** [reply]
Компиляция кода  Ошибка в действии "CS0200" "Property or indexer 'MagickImage.Compression' cannot be assigned to -- it is read only". [Строка: 129; Cтолбец: 5]

**Reply by dlemstra** [reply]
Sorry that should be `image.Settings.Compression` instead.
